### PR TITLE
Create broadcast_text_locale.md

### PR DESCRIPTION
**Pull request by Bogir**

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
links and reference to Trinity might not be very useful to us xD

File: docs/broadcast_text_locale.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The localized text that the male creature will broadcast, or male lpayers can read from gossip menu.
+
+### FemaleText
+
+The localized text that the female creature will broadcast, or female players can read from gossip menu.
+
+#### WDBVerified
+
+This field is used by the TrinityDB Team to determine whether a template has been verified from WDB files (ADB files for this one).
```
Comment (BarbzYHOOL) [inline docs/broadcast_text_locale.md]: here

File: docs/broadcast_text_locale.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+If value is 0 then it has not been parsed yet.
+
+If value is above 0 then it has been parsed with WDB files from that specific client build.
+
+If value is -1 then it is just a place holder until proper data are found on WDBs.
+
+If value is [-Client Build](http://archive.trinitycore.info/DB:Auth:realmlist#gamebuild "DB:Auth:realmlist") then it was parsed with WDB files from that specific [client build](http://archive.trinitycore.info/DB:Auth:realmlist#gamebuild "DB:Auth:realmlist") and manually edited later for some special necessity.
```
Comment (BarbzYHOOL) [inline docs/broadcast_text_locale.md]: wrong links here

**Comment by Helias** [comment]
This is the current table structure in our db:
immagine

Apply these edits:

WDBVerified -> VerifiedBuild

**Default values:**
ID & Verified Build -> 0
locale, MaleText and FemaleText -> NULL

What do you mean with "signed/unsigned"? because about the var type only ID is unsigned.

Anyway good job for all your PRs on wiki, thank you ;-) :muscle: :+1:

**Comment by BarbzYHOOL** [comment]
ty :)

**Pull request merged**